src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
Perhaps it would be better to implement a getter function for this rather than relying on a protected access modifier, as it wouldn't be effective when packages are created at the `A-Packages` stage.